### ISSUE TITLE
Docker module fails with SSLError on OS X running kitematic / boot2docker

### ISSUE THREAD

**Issue by iby** [ansible/ansible-modules-core#1711]
##### ISSUE TYPE

Bug Report
##### COMPONENT NAME

_docker module
##### ANSIBLE VERSION

N/A
##### SUMMARY

I'm trying to start a docker container locally on OS X via boot2docker that was installed via kitematic. My environment contains `DOCKER_HOST` and `DOCKER_CERT_PATH` and `DOCKER_HOST` variables, so I can run docker commands locally w/o any problems. As the matter of fact I can start the container with Ansible using plain `docker run` command, but not the docker module, which gives me this:

> failed: [node01] => {"error": "SSLError(SSLError(CertificateError('no appropriate commonName or subjectAltName fields were found',),),)", "failed": true}

I've seen a few other issues and solutions, namely https://github.com/docker/compose/issues/890#issuecomment-119158667 and https://github.com/ansible/ansible-modules-core/issues/657#issuecomment-80875147, but neither helped. 

Am I missing something obvious? Any thoughts?

**Comment by bcoca** [comment]
EDIT: hiding boilerplate -jctanner

**Comment by giseburt** [comment]
The only workaround I've found is to use `use_tls: "encrypt"`, which isn't supported by `docker_image` yet. (I'm working on a PR now.)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by chouseknecht** [comment]
wontfix

The new docker modules support TLS.
